src/Device.ts

```diff
@@ -102,6 +102,11 @@ export function detectDevice(): BuiltinHandlerName | undefined
 		{
 			return 'Firefox60';
 		}
+		// Firefox on iOS.
+		else if (browser.satisfies({ ios: { OS: '>=14.3', firefox: '>=30.0' }}))
+		{
+			return 'Safari12';
+		}
```

Comment:
Can you please enter in https://bowser-js.github.io/bowser-online/ with that browser and paste here the result?

Reply:
I've done it. Merging. Thanks.

Reply:
Thanks!